packages/rest/src/types.ts

```diff
@@ -100,6 +100,14 @@ export type AjvErrorOptions = {
   singleError?: boolean;
 };
 
+export type AjvCustomFormat = {
+  [name: string]: ajv.FormatValidator | ajv.FormatDefinition;
```

Comment:
The purpose is to support custom ajv formats, however, since ajv supports async validators, I made necessary changes to make it work.

Now I understand that it comes up as a breaking change.

Reply:
During a more detailed search, I found an option `formats` that allows providing custom formats in a similar way I have implemented with `addCustomFormats`. Silly me to not notice it earlier.

Now the point remains on async nature. Since ajv allows custom formats with default options, and they can be async in nature as well, should those be allowed or not?

Reply:
Agreed.